target/linux/ath79/base-files/etc/board.d/02_network

```diff
@@ -34,6 +34,11 @@ ath79_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"1:lan" "2:lan" "3:lan" "4:lan" "0:wan" "5@eth0"
 		;;
+	"tplink,tl-wr841n-v9")
+		ucidef_set_interfaces_lan_wan "eth1.1" "eth0"
```

Comment:
ucidef_add_switch should already set the lan interface. A ucidef_set_interface_wan "eth0" should be sufficient here.